ush/config_defaults.yaml

```diff
@@ -2372,21 +2383,331 @@ global:
+# Flag turning on/off AQM dust option in AQM_RC
+#
+# AQM_DUST_DIR:
+# Path to the directory containing AQM dust file
+#
+# AQM_DUST_FILE_PREFIX:
+# Frefix of AQM dust file
+#
+# AQM_DUST_FILE_SUFFIX:
+# Suffix and extension of AQM dust file
+#
+# DO_AQM_CANOPY
+# Flag turning on/off AQM canopy option in AQM_RC
+#
+# AQM_CANOPY_DIR:
+# Path to the directory containing AQM canopy files
+# 
+# AQM_CANOPY_FILE_PREFIX:
+# File name of AQM canopy file
+#
+# AQM_CANOPY_FILE_SUFFIX:
+# Suffix and extension of AQM CANOPY file
+# 
+# AQM_FIRE_DIR:
+# Path to the directory containing AQM fire files
+# 
+# AQM_FIRE_FILE_PREFIX:
+# Prefix of AQM FIRE file
+# 
+# AQM_FIRE_FILE_SUFFIX:
+# Suffix and extension of AQM FIRE file
+#
+# AQM_RC_FIRE_FREQUENCY:
+# Fire frequency in aqm.rc
+# 
+# DO_AQM_PRODUCT
+# Flag turning on/off AQM output products in AQM_RC
+#
+# AQM_RC_PRODUCT_FN:
+# File name of AQM output products
+#
+# AQM_RC_PRODUCT_FREQUENCY:
+# Frequency of AQM output products
+#
+# AQM_LBCS_DIR:
+# Path to the directory containing chemical LBC files
+# 
+# AQM_LBCS_FILES:
+# File name of chemical LBCs
+#
+# AQM_GEFS_DIR:
+# Path to the directory containing GEFS aerosol LBC files
+#
+# AQM_GEFS_FILE_PREFIX:
+# Prefix of AQM GEFS file ("geaer" or "gfs")
+#
+# AQM_GEFS_FILE_CYC:
+# Cycle of the GEFS aerosol LBC files only if it is fixed
+# 
+# DO_AQM_CHEM_LBCS:
+# Add chemical LBCs to chemical LBCs
+# 
+# DO_AQM_GEFS_LBCS:
+# Add GEFS aerosol LBCs to chemical LBCs
+# 
+# NEXUS_INPUT_DIR:
+# Same as GRID_DIR but for the the air quality emission generation task.
+# Should be blank for the default value specified in setup.sh
+# 
+# NEXUS_FIX_DIR:
+# Directory containing grid_spec files as the input file of nexus
+# 
+# NEXUS_GRID_FN:
+# File name of the input grid_spec file of nexus
+#
+# NUM_SPLIT_NEXUS:
+# Number of split nexus emission tasks
+#
+# NEXUS_GFS_SFC_OFFSET_HRS: 0
+# Time offset when retrieving gfs surface data files
+# 
+# NEXUS_GFS_SFC_DIR:
+# Path to directory containing GFS surface data files
+# This is set to COMINgfs when DO_REAL_TIME=TRUE. 
+#
+# NEXUS_GFS_SFC_ARCHV_DIR: 
+# Path to archive directory for gfs surface files on HPSS
+#
+# PT_SRC_BASEDIR:
+# Parent directory containing point source files for CONUS/AK/HI
+#
+# COLDSTART:
+# Flag turning on/off warm start
+#
+# WARMSTART_CYCLE_DIR:
+# Path to the directory where RESTART dir is located for warm start
+#
+# AQM_AIRNOW_HIST_DIR:
+# Path to the directory where the historical AIRNOW data are located
+#
+# COMINairnow:
+# Path to the real-time AIRNOW observation data
+#
+# COMINgfs:
+# Path to the real-time GFS data
+#
+# COMINgefs:
+# Path to the real-time GEFS data
+#
+#-----------------------------------------------------------------------
+#
+  AQM_CONFIG_DIR: ""
+  AQM_BIO_DIR: ""
+  AQM_BIO_FILE: "BEIS_SARC401.ncf"
+  DO_AQM_DUST: true
+  AQM_DUST_DIR: "/path/to/dust/dir"
+  AQM_DUST_FILE_PREFIX: "FENGSHA_p8_10km_inputs"
+  AQM_DUST_FILE_SUFFIX: ".nc"
+  DO_AQM_CANOPY: false
```

Comment:
The 'DO_AQM_*' flags are only for Online-CMAQ. So I think it would be better for them to be placed in cpl_aqm_parm. Currently, other 'DO_*' flags are not in the workflow section but in the 'global' section.